### PR TITLE
Updated qpc copr repo locaton in Jenkinsfile

### PR DESCRIPTION
**Pull request by himmAllRight**
Updated qpc copr repo locaton in the Jenkinsfile and switched the deprecated 'archive' commands to 'archiveArtifacts'.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -52,7 +52,7 @@ def configureDocker = {
 
 def installQpcClient() {
     sh '''\
-    sudo wget -O /etc/yum.repos.d/chambridge-qpc-fedora-28.repo https://copr.fedorainfracloud.org/coprs/chambridge/qpc/repo/fedora-28/chambridge-qpc-fedora-28.repo
+    sudo wget -O /etc/yum.repos.d/group_quipucords-qpc-fedora-29.repo https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/repo/fedora-28/group_quipucords-qpc-fedora28.repo
```
Comment (himmAllRight) [inline Jenkinsfile]: Could/should this be conditional?

Comment (elyezer) [inline Jenkinsfile]: We are using Fedora 28 to build this, so the repo should be the one for that OS.

Comment (elyezer) [reply Jenkinsfile]: Right now we are using a single OS to build and test, if we start testing on more OSes then we should add a conditional to properly select the repo.

Comment (himmAllRight) [reply Jenkinsfile]: Got it. I was unsure about which to use.

Comment (himmAllRight) [reply Jenkinsfile]: actually looking at this... it is using 28, but pulling it down to a file named 29. I'll fix that...

**Pull request merged**